components/director/internal/domain/root_resolver.go

```diff
@@ -250,6 +251,14 @@ func (r *queryResolver) Tenants(ctx context.Context) ([]*graphql.Tenant, error)
 	return r.tenant.Tenants(ctx)
 }
 
+func (r *queryResolver) AutomaticScenarioAssignment(ctx context.Context, scenarioName string) (*graphql.AutomaticScenarioAssignment, error) {
+	return nil, errors.New("not implemented")
```

Comment:
Maybe it would be good to return some mock data instead of `not implemented` error? Open question

Reply:
I would go with the implementation of the mocks in the actual scenarios assignment domain layer (in the scenarios assignment dedicated resolver)